### ISSUE TITLE
  This model version already exists at None even when overwrite duplicates.

### ISSUE THREAD

**Issue by maniac-0s** [butaixianran/Stable-Diffusion-Webui-Civitai-Helper#310]
When you download a model and then decide to download it again, for instance you decided you want to download the training files too, it tells you that the file already exists, even if you select overwrite for duplicate files.

When you then delete the model file and try to download again, it tells you "This model version already exists at None"

**Comment by butaixianran** [comment]
I don't think you are using this extension, you could be using another fork of this extension, which may has same extension name. 

In this extension, there is no error msg as "This model version already exists at xxxxx".
